debian/control

```diff
@@ -1,53 +1,19 @@
 Source: pop-plymouth-theme
 Section: x11
 Priority: optional
-Maintainer: Jeremy Soller <jeremy@system76.com>
-Uploaders: Ian Santopietro <ian@system76.com>
-Build-Depends: debhelper (>= 10), imagemagick
-Standards-Version: 3.9.8
-Homepage: https://github.com/system76/pop-plymouth-theme
+Maintainer: 13r0ck <bnr@tuta.io>
+Build-Depends: debhelper (>= 10.5.1)
+Standards-Version: 4.5.0
 
-Package: plymouth-theme-pop-basic
+Package: pop-plymouth-theme
 Architecture: all
-Depends: ${misc:Depends}, plymouth, plymouth-label, plymouth-themes, pop-fonts
+Depends: ${misc:Depends}, plymouth, plymouth-label
```

Comment:
The file `/usr/lib/x86_64-linux-gnu/plymouth/two-step.so` is provided by the package `plymouth-theme-spinner` (which is also a dependency of `plymouth-themes`).

I haven't tested it, but I'm guessing that's required here.

Reply:
Ah yes you are right, I thought my issue with `plymouth-theme-spinner` was from this, but it is actually from the [plymouth fork i made](https://github.com/pop-os/plymouth/blob/re-write/debian/control#L108).

I am kinda stuck though. It looks like the [control file](https://github.com/pop-os/plymouth/blob/re-write/debian/control#L108) should conflict and remove the old version of `plymouth-theme-spinner` and install `plymouth-themes`, but it doesn't. Do you have any idea what is going on there?

Reply:
More specifically I am getting the error:
`trying to overwrite '/usr/lib/x86_64-linux-gnu/plymouth/two-step.so', which is also in package plymouth-theme-spinner 0.9.4git20200323-0ubuntu6.2`

Reply:
Sorry, I'm still not getting it. Any advice?
I tried every combination of plymouth-theme(s)-spinner and I still can't get it to work.

I can get it all working with
1. [modify pop-os/default-settings](https://github.com/pop-os/default-settings/pull/129/commits/2c33ab7f07c9ec9189592a9e8aca07501a1987c3#)
2. apt full-upgrade
3. uninstall `plymouth-theme-spinner`
4. install fonts-cantarell
5. install pop-os/plymouth
6. install pop-os/plymouth-them (`re-write`) (but without depending on `plymouth-theme-spinner`)

Reply:
It looks like Ubuntu has `plymouth-theme-spinner` as a separate package but Debian does not. I don't know if there's VCS for Ubuntu's plymouth packaging anywhere (their package just links Debian's git, which sometimes has Ubuntu branches but not here).

You can at least get a `.debian.tar.xz` to compare with at https://packages.ubuntu.com/source/impish/plymouth.

We probably want to match how Ubuntu packages Plymouth so we can cleanly upgrade from their version to ours, and then back to theirs in later releases (so we don't have to package Plymouth forever if Ubuntu already packages an acceptable version).

Reply:
It looks like Ubuntu split that package off in https://git.launchpad.net/ubuntu/+source/plymouth/commit/?id=585f099fc2a26ab5c79857740b9fd4a3b7178300 (the git commits there are kind of odd and auto-generated from something else).